Alamofire.podspec

```diff
@@ -18,4 +22,6 @@ Pod::Spec.new do |s|
   s.source_files = 'Source/**/*.swift'
 
   s.frameworks = 'CFNetwork'
+
+  s.resource_bundles = {'Alamofire' => ['Source/PrivacyInfo.xcprivacy']}
```

Comment:
It does not allow to find the xcprivacy file properly, since this comes under resources folder in xcode. Can we use the generic path like this:
s.resource_bundles = {'Alamofire' => ['Alamofire/*.{xcprivacy}']}